docs/UserGuide.md

```diff
@@ -64,16 +78,25 @@ Views a module stored in the system
   Examples:
    * `viewmodule n/cs2103t` views the specified module
 
-#### Adding a zoom link to a module: `add zoom`
-
-  Adds a zoom link to an existing module.
+#### Adding a zoom link for a specific lesson to a module: `addzoom`
 
-  Format: `add zoom` **_`[MODULE_NAME]`_** **_`[ZOOM_LINK]`_**
+  Adds a zoom link for a specific lesson to an existing module.
 
-  * Adds a zoom link [ZOOM_LINK] to a module named **_`[MODULE_NAME]`_**
-
-  Example of usage:
-  `add zoom cs2103T https://sample.zoom.us` adds a zoom link `https://sample.zoom.us` to the module named `cs2103T`
+  Format: `addzoom INDEX n/LESSON_NAME z/ZOOM_LINK`
+  
+   * Adds a zoom link to the module at the specified `INDEX`. 
+   
+   * The index refers to the index number of the module shown on the displayed module list. 
+   
+   * The index **must be a positive integer** 1, 2, 3...
+   
+   * The zoom link provided must be a link that uses the NUS domain. A typical zoom link that is under the NUS domain
+     would start with: `https://nus-sg.zoom.us/`. Zoom links that do not belong to the NUS domain would not be accepted.
```

Comment:
Just wondering, are we going to allow the user to add a zoomlink without NUS Domain in the future?

Reply:
Yes, this can be an area for consideration. However, I would assume that most if not all zoom links for NUS modules would use the NUS domain. This is based on my experiences with regards to online learning this semester.